diff -up tree-1.6.0/tree.c.args tree-1.6.0/tree.c
--- tree-1.6.0/tree.c.args	2013-04-11 11:53:47.676572649 +0100
+++ tree-1.6.0/tree.c	2013-04-11 11:56:10.962350448 +0100
@@ -536,6 +536,8 @@ void usage(int n)
 	"  --filelimit # Do not descend dirs with more than # files in them.\n"
 	"  --timefmt <f> Print and format time according to the format <f>.\n"
 	"  -o filename   Output to file instead of stdout.\n"
+	"  --du          Print directory sizes.\n"
+	"  --prune       Prune empty directories from the output.\n"
 	"  -------- File options ---------\n"
 	"  -q            Print non-printable characters as '?'.\n"
 	"  -N            Print non-printable characters as is.\n"
